### ISSUE TITLE
Documentation inconsistent with install - minor

### ISSUE THREAD

**Issue by fm75** [jupyterhub/the-littlest-jupyterhub#241]
The install docs insist that after a small delay the install will finish with "Done!" [There is code to do that](https://github.com/jupyterhub/the-littlest-jupyterhub/blob/a9f49f9172803af9909bf3dbfd95f15260d3092c/tljh/installer.py#L441), but I don't think the logger has been configured to report to `stdout`

**Comment by fm75** [comment]
This can be fixed easily.

1. in tljh/log.py it needs the following line at the end of the file
```python
    return logger
```
2. in tljh/installer.py, need to change:
```python
        init_logging()
```
to
```python
        logger = init_logging()
```

I closed a PR on this because it had some of my work on updating documentation. It also had upstream changes showing as changes in the same PR.

**Comment by fm75** [comment]
This was closed with PR #248
